src/herbivoresAndCarnivores.js

```diff
@@ -1,15 +1,41 @@
 'use strict';
 
 class Animal {
+  static alive = [];
   // write your code here
+  constructor(name) {
+    this.name = name;
+    this.health = 100;
+    this.hidden = false;
```

Comment:
The `hidden` property should only be present on instances of the `Herbivore` class. It's not necessary for all `Animal` instances to have this property, as carnivores do not need to hide. Consider moving the `hidden` property initialization to the `Herbivore` constructor.